src/main/java/frc/robot/subsystems/Tower.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+// Subsystem that controls the tower of the robot
+
+public class Tower extends SubsystemBase {
+  /** Creates a new Tower. */
```

Comment:
This javadoc should be above the class definition, and expanded to 3 lines (first and last line blank). Should also be slightly more descriptive (see https://github.com/246overclocked/2022-Robot-Code-Java/blob/main/src/main/java/frc/robot/subsystems/Outtake.java for an example)

Reply:
This javadoc appears to be misplaced still (the one that says `/** Creates a new Tower. */`)